### DISCUSSION TITLE
More informative error messages

### DISCUSSION THREAD

**Discussion by steelahhh** [bumble-tech/appyx#686] [Ideas]
Getting an exception like this, makes it practically impossible to figure out the root cause

```Rendering and children management is out of sync: requested com.bumble.appyx.core.navigation.NavKey@3a63756 but have only [com.bumble.appyx.core.navigation.NavKey@465492c, com.bumble.appyx.core.navigation.NavKey@10560079]```

Would be nice to get more information in this message, e.g.: 
1. more descriptive `NavKey`, mimicking `toString` of a `data class`
2. type and key of navModels that caused this exception

**Comment by KovalevAndrey** [comment]
Hey @steelahhh thanks for the suggestion, It indeed can be improved. PRs are welcome :)
